src/ApparentHorizons/YlmSpherepack.hpp

```diff
@@ -0,0 +1,231 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "YlmSpherepackStorage.hpp"
+
+/// \ingroup Spectral
+/// \brief
+/// C++ interface to SPHEREPACK.
+class YlmSpherepack {
+ public:
+  using first_deriv_t = tnsr::i<DataVector, 2, Frame::Logical>;
+  using second_deriv_t = tnsr::ij<DataVector, 2, Frame::Logical>;
+  using interpolation_info_t = std::vector<std::vector<double>>;
```

Comment:
I'm not sure what our decision is on this, but I generally prefer if we use type aliases like this as opposed to using them as metavariables that they be camel cased.

Reply:
Yes, the `_t` gives it away that these are just types, not metavariables.

Reply:
So FirstDeriv_t ?

Reply:
I made them public because public member functions take objects of these types as arguments or return types, so the user can say e.g. YlmSpherepack::first_deriv_t when declaring them.

Reply:
Just FirstDeriv